source/administration/object-management/object-versioning.rst

```diff
@@ -270,7 +270,7 @@ using an S3-compatible SDK.
         MinIO deployment.
 
       - Replace ``BUCKET`` with the 
-        :mc-cmd:`target bucket <mc version ALIAS>` on which to enable
+        :mc-cmd:`target bucket <mc mb ALIAS>` on which to enable
```

Comment:
Huh. I must be going crazy because I swear the example was for `mc mb` before. Now it's not.

```suggestion
        :mc-cmd:`target bucket <mc version enable ALIAS>` on which to enable
```

Reply:
😂